src/lib/sidenav/sidenav.ts

```diff
@@ -134,15 +143,15 @@ export class MdSidenav implements AfterContentInit {
 
   /** Open this sidenav, and return a Promise that will resolve when it's fully opened (or get
    * rejected if it didn't). */
-  open(): Promise<void> {
+  open(): Promise<boolean> {
```

Comment:
Rather than emitting a `boolean` directly, can we create an event object and include this information? That lets us add additional information to the event in the future.